src/content/docs/logs/ui-data/timestamp-support.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Understanding and using timestamps in logs
+tags:
+  - Logs
+  - Timestamp
+  - Timestamps
+  - Forwarding Logs
+metaDescription: "How New Relic handles timestamps in logs."
+---
+
+## Introduction [#introduction]
+
+One of the most important attributes in a log event is the **timestamp**.  It is the exact moment in time that tells when the event happened and is critical for ordering events in chronological order as well as for searching logs in specific timeframes.  The most basic question all logs answer is **what happened, and when** it happened.  In most cases, the timestamp will appear somewhere in the beginning of the log event, but this is not always the case.  In some cases, it may appear much later in the log.  In logs that are in a JSON format they will usually contain some attribute that identifies the timestamp, like `timestamp` or `log_timestamp`, `time`, etc.  Some logs may not contain timestamps at all, and in these cases New Relic will always assign a timestamp automatically to the event at the time it is ingested into the system.  New Relic also indexes the timestamp of each log, since they are crucial for efficient querying.
+
+There are many, many different timestamp formats as you can see in some of these example logs below.  There is no single standard for the format of a timestamp.  They may or may not contain certain fields like the date, ms precision, or the timezone. 
+
+```
+time="2023-11-16T14:50:44Z" level=info msg="checking pixie-operator.v0.1.4"
```

Comment:
@marcelschlapfer Do we need to keep this example? I'm reworking the structure and I think the content works better without this particular example, but I won't remove it if your team thinks it's necessary for the integrity of the doc.

Reply:
You can remove it.  I just added several examples from logs we had in our test platform.  The point can still be made with 4 examples vs 5.